ai_serving/fasttext_clf/clf.json

```diff
@@ -0,0 +1,1076 @@
+{
```

Comment:
Rename this file to be training_data or something similar. Just want to make it clear this is the training data.